geometry/grassmann.hpp

```diff
@@ -288,6 +288,82 @@ Multivector<Quotient<LScalar, RScalar>, Frame, rank>
 operator/(Multivector<LScalar, Frame, rank> const& left,
           RScalar const& right);
 
+template<typename LScalar,
```

Comment:
Please don't trust `clang-format` for template parameter lists, it just puts too much vertical space; follow everywhere the style of lines 285-286.